test_runner/src/main/kotlin/ftl/gc/GcAndroidTestMatrix.kt

```diff
@@ -28,6 +29,15 @@ object GcAndroidTestMatrix {
         value = this@toEnvironmentVariable.value
     }
 
+    @VisibleForTesting
+    internal fun TestSetup.setEnvironmentVariables(args: AndroidArgs, testConfig: AndroidTestConfig) = this.apply {
```

Comment:
This extension deserves for dedicated file. Check out `setupAndroidTest` for example.